test_scripts/API/SubtleAlertStyle/SubtleAlert/009_GetSystemCapabilities_no_app_specific_capabilities.lua

```diff
@@ -0,0 +1,77 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
I would propose to move this new script to `https://github.com/smartdevicelink/sdl_atf_test_scripts/tree/develop/test_scripts/WidgetSupport/Capabilities` and use `test_scripts/WidgetSupport/common` module.
It has required functions for data generation, e.g. `common.getOnSystemCapabilityParams()` which can be easily modified to include new `subtleAlert` text fields.
And therefore include this script into `widget_support.txt` test set.